saphir/src/file/middleware.rs

```diff
@@ -134,10 +153,9 @@ impl FileMiddleware {
 
         builder = builder
             .header(http::header::ACCEPT_RANGES, "bytes")
-            .header(header::CONTENT_TYPE, mime_type.to_string())
+            .header(header::CONTENT_TYPE, Self::guess_path_mime(&path).to_string())
             .header(header::CONTENT_LENGTH, size)
-            .header(header::CACHE_CONTROL, "public")
-            .header(header::CACHE_CONTROL, "max-age=0")
+            .header(header::CACHE_CONTROL, "public, max-age=86400")
```

Comment:
This should actually be configurable. Existing combo was weird because we specified public cache but a max-age of zero, disabling caching. We should provide it as a config and not just assume a default.

Reply:
specifying public but setting max-age 0 is what we wanted in our use case. We want the browser to verify each time it has to access a file, but we want the ETAG to be cached in any cache whenever it is possible.
But indeed it should be configurable, the default should be public with max-age 0